src/components/VitalsSideBar.vue

```diff
@@ -68,7 +40,19 @@ export default {
     return{
       showHide: true
     }
-  },  
+  },
+  computed: {
```

Comment:
toto sa da prakticky robit hore... a ja som to vlastne aj robil hore (predtym)

Reply:
imho to patri sem do tejto komponenty

Reply:
mmmmmmmm ale vlastne asi budes potrebovat  ref ked to urobis o lvl hore, leda ze by si to konstruoval v tom priradeni... to by mohlo ist